hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestMergesSplitsAddToTracker.java

```diff
@@ -228,7 +228,7 @@ private void validateDaughterRegionsFiles(HRegion region, String orignalFileName
     List<StoreFileInfo> infos = region.getRegionFileSystem().getStoreFiles("info");
     final MutableBoolean foundLink = new MutableBoolean(false);
     infos.stream().forEach(i -> {
-      i.getActiveFileName().contains(orignalFileName);
+      assertTrue(i.getActiveFileName().contains(orignalFileName));
       if(i.getActiveFileName().contains(untrackedFile)){
```

Comment:
assertFalse?

Or both lines could be rewrite as assertThat?

Reply:
this change is incorrect. pushing a fix.